<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>110</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
 background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                             stop: 0 #5f2c82,
                             stop: 1 #49a09d);
  color: white;
  padding: 20px;
  border-radius: 10px;
 font: bold 20pt &quot;Arial&quot;;
 }</string>
      </property>
      <property name="text">
       <string>Inventory and Production Management System</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>430</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="panel_dashboard">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <widget class="QPushButton" name="btn_materials">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>190</y>
            <width>221</width>
            <height>40</height>
           </rect>
          </property>
          <property name="text">
           <string>Materials</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../assets/Images/IMG-Materials.png</normaloff>../assets/Images/IMG-Materials.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_users">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>221</width>
            <height>40</height>
           </rect>
          </property>
          <property name="text">
           <string>Users</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../assets/Images/IMG-SettingsUser.png</normaloff>../assets/Images/IMG-SettingsUser.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="btn_requests">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>140</y>
            <width>221</width>
            <height>40</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Requests</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../assets/Images/IMG-Requests.png</normaloff>../assets/Images/IMG-Requests.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QMdiArea" name="mdi_container"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1520</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
